docs/user-guide/binned-data/binned-data.ipynb

```diff
@@ -292,8 +292,8 @@
     "s0 = binned['x', 0]\n",
     "s1 = binned['x', 1]\n",
     "print(f'total events: {sc.sum(binned.bins.sum().data)}')\n",
-    "print(f'events x=0:  {sc.sum(s0.bins.sum().data)}')\n",
-    "print(f'events x=1:  {sc.sum(s1.bins.sum().data)}')"
```

Comment:
When I was going through this notebook there seemed to be a couple of matplotlib figures which were created but then not displayed in code cells 5 and 8. Are these figures supposed to be displayed? If so I found adding fig.show() as the final line of the cell resolved things.

Reply:
I had the problem when running the notebook locally. Are you referring to the docs pages or the notebooks?

Reply:
Ah, you are right the plots have disappeared. @nvaytet when did this break? Any ideas?

Reply:
I have the same problem. matplotlib figures are only shown when you use `%matplotlib inline`. So vanilla matplotlib and scipp widgets are incaompatible at the moment.